tools/devcontainer

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "json"
+require "open3"
+
+VARS = {
+  "localWorkspaceFolderBasename" => Dir.pwd.split(File::SEPARATOR).last
+}
+
+data = File.binread ".devcontainer/devcontainer.json"
+data.gsub!(/^\s*\/\/.*$/, "") # strip comments
+info = JSON.load data
+
+working_dir = info["workspaceFolder"].gsub(/\${([^}]*)}/) { |x|
+  VARS.fetch($1)
+}
+
+env = info["containerEnv"].map { |k, v| "-e #{k}=#{v}" }.join " "
+
+case ARGV[0]
+when "up"
+  system "docker compose -f .devcontainer/#{info["dockerComposeFile"]} up"
```

Comment:
```suggestion
  system "docker", "compose", "-f", ".devcontainer/#{info["dockerComposeFile"]}", "up"
```

This way you don't need to concern yourself with spaces inside the `dockerComposeFile` path.